dd-java-agent/agent-debugger/src/main/java/com/datadog/debugger/probe/SpanDecorationProbe.java

```
@@ -150,6 +151,13 @@ public void evaluate(CapturedContext context, CapturedContext.Status status) {
           if (!condition) {
             continue;
           }
+        } catch (PermanentEvaluationException ex) {
+          LOGGER.debug("Permanent evaluation error: ", ex);
```

SpanTag can have multiple conditions and therefore multiple errors. I think in those cases we should collect all errors and eval both diagnostic message + eval errors on the span. currently permanent errors are not reported as errors when looking at the span